tests_metricflow/dataflow/optimizer/test_predicate_pushdown_optimizer.py

```diff
@@ -325,6 +325,7 @@ def test_aggregate_output_join_metric_predicate_pushdown(
     )
 
 
+@pytest.mark.skip("Predicate pushdown is not implemented for some of the nodes in this plan")
```

Comment:
were these passing before this change 🤔

Reply:
Yes! The optimizers weren't correctly going through all the parent nodes, so this was never hitting the `AliasSpecsNode`. That node is new and doesn't have an implementation for the pushdown optimizer.